components/ledger/internal/domain/onboarding/organization/organization.go

```diff
@@ -2,8 +2,9 @@ package organization
 
 import (
 	"database/sql"
-	"github.com/LerianStudio/midaz/common"
 	"time"
+
+	"github.com/LerianStudio/midaz/common"
```

Comment:
Please don't touch the structs in the ledger for now; there's a ticket to handle that.